lib/proxy/plgdata.rb

```diff
@@ -8,6 +8,12 @@ def initialize(proxy)
       super(proxy, Rails.configuration.constants['plgdata'])
     end
 
+    def rewrite_env(env)
+      super.tap do |e|
+        e['HTTP_ACCEPT'] = "*/*" if e['PATH_INFO'].start_with?('/upload')
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.